### ISSUE TITLE
[Bug] Water in Leather Waterskin cannot be purified

### ISSUE THREAD

**Issue by IdrisQe** [TeamMoegMC/TheWinterRescue#47]
### Error Behaviour

In the quest book, it says that you can boil-purify water as soon as you get a campfire, but the waterskin filled with dirty water cannot be placed on a campfire to boil.

### Expected Behaviour

The waterskin should be able to be boiled to get purified water.

### Reproduce

N/A

### Modpack version

0.5.1-hf5

**Comment by yuesha-yc** [comment]
This is intended. use cups or the stew pot.

**Comment by IdrisQe** [comment]
Which requires wooden planks, which requires a Tree Stump from the Stone Age Mod, which requires 3 Vanilla Stone blocks, which need to be made from Cobblestone via a Pottery Kiln?

Just to get purified water? Your initial waterskin is supposed to last that long?

**Comment by yuesha-yc** [comment]
Yes. Doesn't require any metal so it's easy right? Also don't forget you
can just drink normal water if necessary.

**Comment by IdrisQe** [comment]
Not easy in the slightest, no, especially with the cluttered quest book. It took us hours to even get to the stone age section, especially because of the blizzard that literally happened from the second day all the way for the next like... entire real life hour. And we spawned in the middle of a tiny island on a frozen ocean biome.

Speaking of normal water, why can't you fill a cup or waterskin with snow? And why can't you fill them from flowing water since it doesn't use the source block up anyway.

**Comment by yuesha-yc** [comment]
I still reject allowing you to boil water in your leather sack because 1. the leather will catch fire 2. it diminishes the use of stew pot. 
I accept some of your suggestions and some of them will be in game in 0.5.2: 1. there will be time pause when you are reading the quest book, saving you enough time before blizzard 2. i can try to add a recipe for melting snowball for water in a barrel, great suggestion. 3. pottery kiln and charcoal pit will have their ponder scenes to tutor you.
